posts/2025-12-23-investigating-spirv-for-the-shader-system-part-3.md

```diff
@@ -0,0 +1,343 @@
+---
+title: "Investigating SPIR-V for the shader system - Part 3"
+author: youness
+popular: false
+image: /images/spir.png
+tags: ['.NET', 'Shaders']
+---
+In this third part we're going to design a parser for the SDSL language!
+---
+
+Table of Contents:
+
+[[TOC]]
+
+So many things happened since the [the last blog post](/blog/investigating-spirv-for-the-shader-system-part-2/) but none were written in a blog post format, fortunately for me, with the holidays I have time to tell more about it.
```

Comment:
This can be also reworded, so you can link also to the **part 1** of the article. Maybe you could write 1-2 (or more) sentences about the first part, and include a link, and the same for the second part. Or maybe your **Preface** section is better to remind and link part 1 and part 2?